docs/library/esp32.rst

```diff
@@ -114,6 +114,11 @@ methods to enable over-the-air (OTA) updates.
 
     Sets the partition as the boot partition.
 
+    **Note:** do not enter deepsleep (:func:`machine.deepsleep`) after changing
```

Comment:
```suggestion
   .. note:: Do not enter deepsleep (:func:`machine.deepsleep`) after changing
```

Just a nitpick as it's possible to have Sphinx highlight the note. Thanks for documenting the extra limitation!

Reply:
Fixed. Thanks - that's better.